### ISSUE TITLE
Accessibility: skipnav missing main-content target

### ISSUE THREAD

**Issue by arcepaul** [NCIOCPL/cgov-digital-platform#2225]
The site skipnav does not work because it is missing the 'main-content' id.  This problem exists in production on cgov.

Add the main-content id to main.

**Comment by markecramer** [comment]
Paul, thanks for making the skip nav visible on active state.

Skipnav looks fine to me, its bypassing the navigation.

**Comment by bennettcc** [comment]
MVP1 launched yesterday - ticket no longer needed
